big_tests/run_common_test.erl

```diff
@@ -588,3 +590,84 @@ handle_file_error(_FileName, Other) ->
     Other.
 
 %% ------------------------------------------------------------------
+
+%% cover_server process is using io:format too much.
+%% This code removes duplicate io:formats.
+%% Each io:format would be written only once with this group leader.
+%%
+%% Example of a message we want to write only once:
+%% "Analysis includes data from imported files" from cover.erl in Erlang/R19
+%% ------------------------------------------------------------------
+
+%% Set a new group leader for cover_server
+deduplicate_cover_server_console_prints() ->
```

Comment:
I'd like to see this functionality moved to a separate module.

Reply:
separated.